src/DynamicRupture/Misc.h

```diff
@@ -26,6 +28,19 @@ constexpr F forEach(TupleT&& tuple, F&& functor) {
       std::forward<F>(functor),
       std::make_index_sequence<std::tuple_size<std::remove_reference_t<TupleT>>::value>{});
 }
+
+template <size_t exp>
+real power(real base) {
```

Comment:
is there a reason why this is not in Misc.cpp?

Reply:
Yes templates have to be specified in headers, only template specializations (in this case for `exp=8` can be put into a cpp file.

Reply:
ok good to know!